common/src/main/java/com/example/android/uamp/media/library/BrowseTree.kt

```diff
@@ -106,6 +114,10 @@ class BrowseTree(context: Context, musicSource: MusicSource) {
                 recommendedChildren += mediaItem
                 mediaIdToChildren[UAMP_RECOMMENDED_ROOT] = recommendedChildren
             }
+            // Add one media item as the "most recently listened media item"
```

Comment:
I'm finding that most of the time the static media controls aren't created because inside `onLoadChildren` the mediaSource fails to load. I believe this is because `mediaSource` is never initalized because `MusicService::onCreate` is never called.

Reply:
I'll be more specific: media resumption doesn't work after a reboot because `mediaSource` hasn't been initialized.

Reply:
We may need to save the recent item album art and metadata to disk, then load it as soon as `onLoadChildren` is called.